pkg/probes/package_probes.go

```diff
@@ -9,6 +9,6 @@ type Probe interface {
 	GetMachineImageID(platformType string, cpuArch cpu.Architecture, region string) (string, error)
 	GetStartingToken() string
 	GetEndingToken() string
-	GetExpandedUserData(map[string]string) (string, error)
+	GetExpandedUserData(map[string]string, string) (string, error)
```

Comment:
Minor - this will help us remember what these values are supposed to be in the future if we ever implement a new probe.
```suggestion
	GetExpandedUserData(userDataVariables map[string]string, userDataTemplate string) (string, error)
```